cli/winresources/README.md

```diff
@@ -1,7 +1,7 @@
 ## Generate `event_messages.bin`
 
 ```console
-$ docker run --rm -it -v "$(pwd):/winresources" debian:bullseye bash
+$ docker run --rm -it -v "$(pwd):/winresources" debian:bookworm bash
```

Comment:
Should we update this one to `-slim` too while we're at it, for consistency? (I think we're now using `-slim` pretty much everywhere else :sweat_smile:)

Reply:
Ah, yeah, I'm all for "slim" where we can!

Reply:
updated; we should probably look at having a dockerfile for this, and to make sure it's verified as part of the "generated files", which made me recall we still want to move this package as well;

- https://github.com/moby/moby/issues/44641